utils/modular_model_converter.py

```diff
@@ -1658,19 +1659,24 @@ def convert_modular_file(modular_file):
 
 def save_modeling_file(modular_file, converted_file):
     for file_type in converted_file.keys():
+        file_name_prefix = file_type.split("*")[0]
+        file_name_suffix = file_type.split("*")[-1] if "*" in file_type else ""
+        new_file_name = modular_file.replace("modular_", f"{file_name_prefix}_").replace(
+            ".py", f"{file_name_suffix}.py"
+        )
```

Comment:
what is enabled by this? 🤗

Reply:
Unlike other modeling files, fast image processing files have a suffix (_fast), so this is a way to account for that when writing or reading those files. It's also why I defined the file type for fast image processors as `image_processing*_fast `[here](https://github.com/huggingface/transformers/blob/41fa0fbf03498fe6bb2fefd7812347043d749098/utils/modular_model_converter.py#L1062)